__tests__/server.test.js

```diff
@@ -12,45 +13,40 @@ beforeAll(async () => {
 });
 
 
+//NUKES THE TEST TO END
 afterAll( async () =>{
   await sequelizeDatabase.drop();
 } );
 
 describe('Server', () => {
+  //test for proof of life path
   test('handles root path', async () => {
     const response = await mockRequest.get('/');
-    
+
     expect(response.status).toBe(200);
     expect(response.text).toEqual('I AM ALIVE!');
   });
-  
-  test('describes food router', async () => {
-    const response = (await request.post('/food')).send({
-      name: 'Test',
-      category: 'fruit',
-    });
+
+  test('handles food router', async () => {
+    const response = await request.get('/food');
 
     expect(response.status).toEqual(200);
-    expect(response.body.name).toEqual('Test');
-    expect(response.body.category).toEqual('fruit');
+    expect(response.body.name).toEqual('test');
+    expect(response.body.category).toEqual('test');
   });
-  
+
   test('handle ingredient router', async () => {
-    const response = await request.post('/ingredients').send({
-      name: 'Test',
-      quantity: 1,
-      foodId: 1,
-    });
-    
+    const response = await request.get('/ingredients');
+
     expect(response.status).toEqual(200);
-    expect(response.body.name).toEqual('Test');
-    expect(response.body.quantity).toEqual(1);
-    expect(response.body.foodId).toEqual(1);
```

Comment:
Change it back to before because your test the ingredient route, which has different information than the food route.